nix/packages/ghciwatch.nix

```diff
@@ -90,6 +103,81 @@
       inherit cargoArtifacts;
     };
 
+  releaseArgs =
+    commonArgs
+    // {
+      # Don't run tests; we'll do that in a separate derivation.
+      # This will allow people to install and depend on `ghciwatch`
+      # without downloading a half dozen different versions of GHC.
+      doCheck = false;
```

Comment:
In principle if the build output doesn't include any references to these GHCs then they wouldn't be included as runtime dependencies.  Specifically, if `ghciwatch` is built, tested, and cached then pulling down the cache product for `ghciwatch` won't pull down the accompanying GHCs.

I don't feel that strongly about this, though.